src/components/PrintButton/SinglePrintButtonForPickSlip.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+import { PrintButton, PrintContent } from '../index';
+import { getSelectedSlipData, isPrintable } from '../../utils';
+
+
+class SinglePrintButtonForPickSlip extends React.Component {
```

Comment:
Could you create functional component instead of class?
I do not see any reason to use class.

Reply:
I followed same pattern as existing PrintButton class. You reckon benefit of class outweigh the benefit of consistency of code?

Reply:
I don't mind to use class components but functional components have some benefits, for example after bundling functional component will have smaller bundle size. So, I recommend and React docs recommends to use functional component if it is possible. But you can make your own choice.

Reply:
Agree. it is now changed to functional component.